LabelsRandomColors2.0/src/com/company/Main.java

```diff
@@ -61,6 +61,7 @@ public void mousePressed(MouseEvent e) {
             /** in debugging, quando arriva alla riga successiva, esce dal debugging, ma continua

```

Comment:
A me una eccezione la da': 

```
Exception in thread "AWT-EventQueue-0" java.lang.NumberFormatException: For input string: "ORANGE"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
```

Se usi il debugger, c'e' un menu' che si chiama `Console` dove ci sono eventuali eccezioni.

Data la stack trace sopra, come fai a capire cosa e' andato storto?

Reply:
Da mia interspretazione e cercando su Google, mi verrebbe da dire che il metodo che ho usato si aspetta un numero in esadecimale, ma io gli do una stringa.